apps/omstillingsstoenad-ui/src/components/felles/rhf/RHFCombobox.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import { FieldPath, FieldValues } from 'react-hook-form/dist/types'
+import { RegisterOptions } from 'react-hook-form/dist/types/validator'
+import { ComboboxProps, UNSAFE_Combobox } from '@navikt/ds-react'
+import { useTranslation } from 'react-i18next'
+import { Controller, FieldError, useFormContext } from 'react-hook-form'
+import { get } from 'lodash'
+import { getTransKey } from '~utils/translation'
+
+interface RHFProps extends Omit<ComboboxProps, 'name'> {
+    name: FieldPath<FieldValues>
+    rules?: Omit<RegisterOptions<FieldValues, FieldPath<FieldValues>>, 'required'>
+}
+
+export const RHFCombobox = ({ name, label, options, rules, required = true, ...rest }: RHFProps) => {
+    const { t } = useTranslation()
+
+    const {
+        control,
+        formState: { errors },
+    } = useFormContext()
+
+    const error: FieldError = get(errors, name) as FieldError
+    const errorMsg = !!error ? t(getTransKey(error)) : undefined
+    const labelWithOptional = required ? label : `${label} (${t('felles.valgfri')})`
+
+    return (
+        <div id={name}>
```

Comment:
Legg til Box her, så trenger du ikke legge til i resten av filene.

```suggestion
        <Box maxWidth="14rem" id={name}>
```

Reply:
Tenker at det er mer hensiktsmessig å heller legge disse rundt der vi bruker komboboksen for land. Så blir det ikke noe skjult "css" liggende i et felles komponent